Formula/t/t1lib.rb

```diff
@@ -39,4 +39,55 @@ def install
     system "make", "install"
     share.install "Fonts" => "fonts"
   end
+
+  test do
+    # T1_SetString seems to fail on macOS with "(E) T1_SetString(): t1_abort: Reason: unable to fix subpath break?"
+    # https://github.com/Homebrew/homebrew-core/pull/194149#issuecomment-2412940237
+    (testpath/"test.c").write <<~EOS
+      #include <stdio.h>
+      #include <stdlib.h>
+      #include <t1lib.h>
+
+      int main( void)
+      {
+        int i;
+        T1_SetBitmapPad(16);
+
+        if ((T1_InitLib(NO_LOGFILE)==NULL)){
+          fprintf(stderr, "Initialization of t1lib failed\\n");
+          return EXIT_FAILURE;
+        }
+
+        for( i=0; i<T1_GetNoFonts(); i++){
+          printf("FontID=%d, Font=%s\\n", i, T1_GetFontFilePath(i));
+          printf("FontID=%d, Metrics=%s\\n", i, T1_GetAfmFilePath(i));
+          // T1_DumpGlyph(T1_SetString( i, "Test", 0, 0, T1_KERNING, 25.0, NULL));
+        }
+
+        T1_CloseLib();
+        return EXIT_SUCCESS;
+      }
+    EOS
```

Comment:
```suggestion
    (testpath/"test.c").write <<~C
      #include <stdio.h>
      #include <stdlib.h>
      #include <t1lib.h>

      int main( void)
      {
        int i;
        T1_SetBitmapPad(16);

        if ((T1_InitLib(NO_LOGFILE)==NULL)){
          fprintf(stderr, "Initialization of t1lib failed\\n");
          return EXIT_FAILURE;
        }

        for( i=0; i<T1_GetNoFonts(); i++){
          printf("FontID=%d, Font=%s\\n", i, T1_GetFontFilePath(i));
          printf("FontID=%d, Metrics=%s\\n", i, T1_GetAfmFilePath(i));
          // T1_DumpGlyph(T1_SetString( i, "Test", 0, 0, T1_KERNING, 25.0, NULL));
        }

        T1_CloseLib();
        return EXIT_SUCCESS;
      }
    C
```
This makes some editors (and I think maybe GitHub now) do the correct syntax highlighting here.